### ISSUE TITLE
Plug-In Abandoned?

### ISSUE THREAD

**Issue by owellsjr** [UniversalMailer/UniversalMailer#121]
Apple still refuses to fix the Default Font bug (I stopped bothering to send feedback about it after OS X 10.4) which means UniversalMailer is still necessary. But unfortunately the plug-in appears to be abandoned since it was last updated 2 years ago today. Does anyone know of a fork that is still being maintained? Or if someone can update this plug-in to provide macOS Catalina support? Re-installing the plug-in isn't working as it has for the last few years.

**Comment by antjnr** [comment]
I agree.  Have also sent heaps of feedback.  Ridiculous. Does anyone know if there are updates planned for this plug-in?

**Comment by howdytom** [comment]
Also interested a compatible version for macOS Mojave. Is Universal Mailer still in development?
